### PR TITLE
Add ResolutionSubsampler

### PR DESCRIPTION
**Pull request by iejMac**

### CODE REVIEW

File: video2dataset/subsamplers/resolution_subsampler.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class ResolutionSubsampler:
+    """
+    """
+
+    def __init__(self):
+        pass
+
+    def __call__(self, video_bytes, metadata):
+        return video_bytes, metadata, None
```
Comment (rom1504) [inline video2dataset/subsamplers/resolution_subsampler.py:18]: Looks like implementation missing :)

Comment (iejMac) [reply video2dataset/subsamplers/resolution_subsampler.py:18]: working on it!

File: README.md [hunk 2]
```diff
@@ -91,8 +91,8 @@ This module exposes a single function `download` which takes the same arguments
 * **save_additional_columns** list of additional columns to take from the csv/parquet files and save in metadata files (default *None*)
 * **number_sample_per_shard** the number of sample that will be downloaded in one shard (default *10000*)
 * **timeout** maximum time (in seconds) to wait when trying to download an image (default *10*)
-* **video_height** height of video frames
-* **video_width** width of video frames 
+* **video_size** size of video frames
```
Comment (rom1504) [inline README.md:94]: Can you add the default in parentheses?

File: video2dataset/main.py [hunk 3]
```diff
@@ -40,8 +40,8 @@ def video2dataset(
     subjob_size: int = 1000,
     incremental_mode: str = "incremental",
     max_shard_retry: int = 1,
-    video_height: int = 360,
-    video_width: int = 640,
+    video_size: int = 360,
+    strict_resize: bool = True,
```
Comment (rom1504) [inline video2dataset/main.py:44]: Should the default be False ?

**Pull request merged**